internal/hmyapi/net.go

```diff
@@ -9,21 +9,21 @@ import (
 
 // PublicNetAPI offers network related RPC methods
 type PublicNetAPI struct {
-	net       p2p.Host
-	networkID uint64
+	net            p2p.Host
+	networkVersion uint64
 }
 
 // NewPublicNetAPI creates a new net API instance.
-func NewPublicNetAPI(net p2p.Host, networkID uint64) *PublicNetAPI {
-	return &PublicNetAPI{net, networkID}
+func NewPublicNetAPI(net p2p.Host, networkVersion uint64) *PublicNetAPI {
+	return &PublicNetAPI{net, networkVersion}
 }
 
 // PeerCount returns the number of connected peers
 func (s *PublicNetAPI) PeerCount() hexutil.Uint {
 	return hexutil.Uint(s.net.GetPeerCount())
 }
 
-// NetworkID ...
-func (s *PublicNetAPI) NetworkID() string {
-	return fmt.Sprintf("%d", 1) // TODO(ricl): we should add support for network id (https://github.com/ethereum/wiki/wiki/JSON-RPC#net_version)
+// Version returns the network version, i.e. network ID identifying which network we are using
+func (s *PublicNetAPI) Version() string {
```

Comment:
Version here and NetVersion above.

Reply:
Have to make it name "Version" because the API is `net_version`.